### DISCUSSION TITLE
2007/11/13/%E8%A5%BF%E5%AE%89%E4%B9%8B%E8%A1%8C%E7%9A%84%E7%85%A7%E7%89%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#189] [Announcements]
# 2007/11/13/%E8%A5%BF%E5%AE%89%E4%B9%8B%E8%A1%8C%E7%9A%84%E7%85%A7%E7%89%87

虽然在西安的工作计划安排的蛮紧张的，但我还是抽出了一上午的时间，和同事一起去了一趟兵马俑。来西安不去看看兵马俑就太遗憾了。虽然在电视上看过无数次了，但是真的看到两千年前的陶像站在眼前，还是被他们一下子震撼住了。对于两千年前的科技而言，这是项多么浩大的工程啊。而且居然史料中没有记载。

https://blog.qizhen.xyz/2007/11/13/%E8%A5%BF%E5%AE%89%E4%B9%8B%E8%A1%8C%E7%9A%84%E7%85%A7%E7%89%87

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2007年11月16日 21:35 编辑
to 怒放的生命真是巧啊。希望 LabVIEW 用的越来越广泛，你还可以继续使用它。

回复
清扬
2007年11月15日 14:23 编辑
西安最有特色的景观就是大雁塔和兵马佣了~
 
PS: 在西安校园宣讲的时候看到你了,现场投了简历,不过被NI拒了,面试机会都没有.不知道以后还有没有机会用到LabVIEW~~~

回复
磊
2007年11月15日 10:34 编辑
文物的保护堪忧

回复
Mo
2007年11月13日 17:12 编辑
我更新了….來看這報導吧!!

回复
Lu
2007年11月13日 12:05 编辑
nice trip :)

回复

**Comment by ruanqizhen** [comment]
之前博客下的留言：

阿七
2009年11月20日 19:07 编辑
多睡觉，少写程序。

回复
竹贤
2007年11月19日 17:38 编辑
在那个年代，在妈妈肚子里就缺乏营养，缺乏维生素和钙，也就是说先天不足，这是一；婴儿和幼年时代，也是物质贫乏的时代，营养没有跟上，这是二；第三，小时候体质弱，易得病，那时候，常用一种现在淘汰了的药品：四环素和土霉素，这是对儿童的骨骼和牙齿发育十分有害的药品，吃多了，牙齿就不可能好。
现在弥补吧：多吃一点牛奶之类，含钙多，钙又易吸收的食品；保持口腔卫生；常咀嚼口香糖，按摩牙龈，也许对牙齿有利。

回复
Arthur
2007年11月17日 09:05 编辑
我也有蛀牙，补了以后还有后遗症，冷热酸甜都会引起牙疼

回复
磊
2007年11月17日 00:04 编辑
天

回复
Qizhen
2007年11月16日 21:51 编辑
我使用 Google Reader 看别人blog的

回复
俊
2007年11月16日 19:48 编辑
每次都在outlook上看你的bolg，总是在点击打开文章页面回复滴时候页面慢吞吞的然后死翘翘了。靠。
 
想想自己写spaces用live writer，看spaces用outlook，确实很久灭登陆了。
 
今天终于登陆成功，在这里踩上一脚，鼓掌，噢耶～～～～～～～

回复
fang
2007年11月16日 18:29 编辑
有些恐怖的说。。。

回复
yoyo
2007年11月16日 18:25 编辑
跟我一样啊!

回复

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Hongbo
2008年09月24日 00:21 编辑
看来广州的2.6相当的贵啊！

回复
Unknown
2008年04月9日 23:11 编辑
呵呵，我是西安人，大学在西安，研究生也在西安，从来没有遇见类似的情况。下次你要来西安的话，有什么事情可先咨询一下我，一定给你帮忙。呵呵，我也是初学labview大军中的一员，看到你blog上的文章觉得很是敬佩，向你学习！

回复
何
2007年12月17日 16:35 编辑
每次用LabVIEW时遇到困难时，我总是上你的Blog逛，每次总有收获，很想对你说声谢谢，今天看到你的这篇Blog，我想我错过了最好的时机，你在西安的时候，我们很可能已经见过面了。那是那段特别的日子里让我感觉最轻松的一次交流，虽然我的表现很差。我很清楚，那只是我的梦想，能够跟你们面对面的交流，我已经很高兴了，非常感谢你们给我的机会！

回复
半夏
2007年11月13日 09:16 编辑
呵呵，挺详细。我去西安时好象没在意那么多，不过倒确实觉得西安的交通挺乱。

回复
